source/user-manual/capabilities/log-data-collection/how-to-collect-macoslogs.rst

```diff
@@ -14,13 +14,16 @@ macOS unified logging system (ULS) events can be gathered and forwarded to the m
 
 The ULS centralizes the management and storage of the logs across all the system levels, from Apps to the Kernel itself, rather than writing the data to text-based log files. Because of this, it is necessary to use specific methods to retrieve logs from this particular system.
 
-To gather the logs, Wazuh uses the CLI `log` tool which provides an interesting interface for log collection in a filtered way, since all the ULS logs pass through this single system. These parameters let the user configure the ``level`` of messages to be gathered, filter by the log ``type`` and even use a very specific ``predicate`` to filter the logs given their more specific characteristics.
+Wazuh uses the CLI `log` tool and the `syslog` format to gather the logs. This tool provides an interface for log collection in a filtered way, since all the ULS logs pass through this single system. The query parameters in the Wazuh configuration allow the user to:
```

Comment:
```suggestion
Wazuh uses the CLI `log` tool and the `syslog` format to gather the logs. This tool provides an interface for log collection in a filtered way since all the ULS logs pass through this single system. The query parameters in the Wazuh configuration allow the user to:
```